test/integration/commands/commandUtils.ts

```diff
@@ -1,4 +1,52 @@
 import { MatrixClient } from "matrix-bot-sdk";
+import { strict as assert } from "assert";
+import * as crypto from "crypto";
+
+/**
+ * Returns a promise that resolves to the first event replying to the event produced by targetEventThunk.
+ * @param client A MatrixClient that is already in the targetRoom that can be started to listen for the event produced by targetEventThunk.
```

Comment:
"A MatrixClient that is already in the targetRoom. We'll use it to listen for the event produced by `targetEventThunk`."